pipelines/nextflow.config

```
@@ -24,103 +24,107 @@ params {
 }
 
 process {
-    executor = 'lsf'
+    executor = 'slurm'
     queue = 'standard'
     perJobMemLimit = true
-    withLabel: lsf_default {
-        clusterOptions = ' -C0 -M1000 '
-    }
-    withLabel: slurm_default {
-        executor = 'slurm'
-        queue = '1c2m20d'
+    perJobTimeLimit = true
+    withLabel: rc_1Gb {
```

Bearing in mind that this Nextflow config file is expected to be shared among workflows, I'd suggest to include default resource config so that workflows can function even if they don't have an RC label on all processes:
```suggestion
    perJobTimeLimit = true
    
    // default resources
    memory = { 1.GB }
    time = 168.h

    withLabel: rc_1Gb {
```